package/yast2-rmt.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Jul  5 08:52:44 UTC 2018 - wstephenson@suse.com
+
+- version 1.0.2
```

Comment:
Seems like you haven't bumped up the package version, though: https://github.com/SUSE/yast2-rmt/blob/a4921aaed2151e24c2548c20cc76f60caa0a3334/package/yast2-rmt.spec#L20

Is the version bump really needed?